RegTest.java

```diff
@@ -0,0 +1,35 @@
+package kiparo.com;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class RegTest {
+
+    @Test
+    public void IsCarNumberIsBelorussianPrintBY() {
+        Reg regular = new Reg();
+        String result = regular.IsCarNumberIsBelorussianPrint("1223", true);
+        Assert.assertEquals("1223" + "BY", result);
+    }
+
+    @Test
+    public void IsCarNumberIsBelorussianPrintNotBY() {

```

Comment:
@Test
    public void formatShouldAddNumbeWithNoBySuffix() {
         String carNumber = "MP 743 5";
        String expected = carNumber + " NOT BY";
        boolean isBelarusCarNumber = false;

        String actual = Reg.formatCarNumber(carNumber, isBelarusCarNumber);
        Assert.assertEquals(expected, actual);
    }